src/store.rs

```diff
@@ -103,6 +118,14 @@ impl<'a> InnerStoreLocked<'a> {
         }
     }
 
+    pub fn remove_ttl(&mut self, key: &str) {
+        if let Some(value) = self.state.keys.get(key) {
+            if let Some(expires_at) = value.expires_at {
+                self.state.ttls.remove(&(expires_at, key.to_string()));
+            }
+        }
+    }
+
```

Comment:
I think this should also update the self.state.keys removing the TTL?

Reply:
Yes. This was actually causing a bug 🙈